plugins/score-card/sample-data/all.json

```diff
@@ -315,5 +315,321 @@
         ]
       }
     ]
+  },
+  {
+    "systemEntityName": "podcast",
```

Comment:
I copied `audio-playback.json` and added `scoreLabel` everywhere.
The file is used in tests as well.